dev_scripts/env.py

```diff
@@ -17,9 +17,11 @@
 DEFAULT_SHOW_DOCKERFILE = False
 DEFAULT_DOWNLOAD_PYSIDE6 = False
 
+PYSIDE6_VERSION ="6.6.3.1"
 PYSIDE6_RPM = "python3-pyside6-{pyside6_version}-1.fc{fedora_version}.x86_64.rpm"
 PYSIDE6_URL = (
-    "https://packages.freedom.press/yum-tools-prod/dangerzone/f{fedora_version}/%s"
+    #"https://packages.freedom.press/yum-tools-prod/dangerzone/f{fedora_version}/%s"
+    "https://github.com/freedomofpress/yum-tools-prod/raw/db4a1faecc4835c8f38580cb50f53f6d81bcaf40/dangerzone/f{fedora_version}/%s"
```

Comment:
This line has been added to make the GitHub Actions use the packages in https://github.com/freedomofpress/yum-tools-prod/pull/20, and should be removed before merging this PR.

Reply:
This is no longer the case.